.gitignore

```diff
@@ -48,13 +46,17 @@ local.properties
 # Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio, WebStorm and Rider
 # Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
 
+# Typically we'd want to check these in, but we don't have any defined for the team.
+.idea/**/inspectionProfiles
+
 # User-specific stuff
 .idea/**/workspace.xml
 .idea/**/tasks.xml
 .idea/**/usage.statistics.xml
 .idea/**/dictionaries
 .idea/**/shelf
 .idea/**/deploymentTargetSelector.xml
+.idea/**/deploymentTargetDropdown.xml
```

Comment:
I don't think this matches `.idea/deploymentTargetDropdown.xml`. I think it will match `.idea/a/lot/of/folders/deploymentTargetDropdown.xml`

Reply:
[** matches 0 to any directories](https://stackoverflow.com/questions/32604656/what-is-the-glob-character).

Reply:
Then I think you need a capital 'D' at the end, 'DropDown'.

```
rylin@rylin:~/src/automotive-design-compose$ git status
On branch tim
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	.idea/deploymentTargetDropDown.xml
```

Reply:
Should be fixed.